docs/contents/releases/1-20/2/CHANGELOG-v1-20-eks-2.md

```diff
@@ -0,0 +1,39 @@
+# Changelog for v1-20-eks-2
+
+This changelog highlights the changes for [v1-20-eks-2](https://github.com/aws/eks-distro/tree/v1-20-eks-2).
+
+## Version Upgrades
+
+### Kubernetes
+
+Upgraded Kubernetes to [v1.20.7](https://github.com/kubernetes/kubernetes/releases/tag/v1.20.7)
+
+### Base Image
+
+Ungraded base image (Amazon Linux 2) version to include the latest security fixes.
```

Comment:
Upgraded

Reply:
I wasn't kidding that words are hard haha